tracer/test/Datadog.Trace.Security.Unit.Tests/WafErrorsTests.cs

```diff
@@ -54,9 +55,9 @@ public void FileNotFound()
             initResult.Success.Should().BeFalse();
             initResult.FailedToLoadRules.Should().Be(0);
             initResult.LoadedRules.Should().Be(0);
-            initResult.Errors.Should().BeEmpty();
-            initResult.HasErrors.Should().BeFalse();
-            initResult.ErrorMessage.Should().BeNullOrEmpty();
+            initResult.Errors.Should().Equal(new Dictionary<string, object> { { "diagnostics-error", "Waf could not provide diagnostics on rules or diagnostic format is incorrect" } });
```

Comment:
Seemed strange there was no error when the WAF failed to load because a file path was wrong.